src/renderer/components/+workloads-deployments/legacy-store.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { asLegacyGlobalForExtensionApi } from "../../../extensions/as-legacy-globals-for-extension-api/as-legacy-global-object-for-extension-api";
+import deploymentStoreInjectable from "./store.injectable";
+
+/**
+ * @deprecated use `di.inject(deploymentStoreInjectable)` instead
+ */
+export const deploymentStore = asLegacyGlobalForExtensionApi(deploymentStoreInjectable);
```

Comment:
Hmm. I wonder if we are creating new legacy too lightly. We are going to have to deal with all this not eventually, but very soon :/

Reply:
I didn't want this PR to get too much bigger than it already is ;)